en_US/file-transfer/client.md

```diff
@@ -83,3 +83,7 @@ Payload: not used.
 * `checksum`: SHA256 checksum of the entire file. If specified, this value takes priority over the `checksum` field provided in the `init` command.
 
 Upon receiving the `finish` command, EMQX verifies that it has received all the segments necessary to assemble the file. If the file is successfully exported and its checksum is valid, EMQX responds with a success return code (RC). In case of any errors, an appropriate error response is sent.
+
+## Client SDK Sample
+
+Refer to [Python](https://github.com/emqx/emqx-ft/blob/main/src/emqx_ft/main.py) to get the client SDK sample.
```

Comment:
I wouldn't call that an SDK sample. That's just a sample client we used during development.

Reply:
✅